Applications/DisplayProperties/DisplayProperties.cpp

```diff
@@ -0,0 +1,153 @@
+#include <AK/StringBuilder.h>
+#include <LibCore/CDirIterator.h>
+#include <LibGUI/GAction.h>
+#include <LibGUI/GApplication.h>
+#include <LibGUI/GBoxLayout.h>
+#include <LibGUI/GButton.h>
+#include <LibGUI/GDesktop.h>
+#include <LibGUI/GEventLoop.h>
+#include <LibGUI/GFileSystemModel.h>
+#include <LibGUI/GGroupBox.h>
+#include <LibGUI/GListView.h>
+#include <LibGUI/GScrollBar.h>
+#include <LibGUI/GSplitter.h>
+#include <LibGUI/GTabWidget.h>
+#include <LibGUI/GToolBar.h>
+#include <LibGUI/GWidget.h>
+#include <LibGUI/GWindow.h>
+
+#include <Servers/WindowServer/WSWindowManager.h>
+
+#include "DisplayProperties.h"
+#include "ItemListModel.h"
+
+DisplayPropertiesWidget::DisplayPropertiesWidget()
+    : m_wm_config(CConfigFile::get_for_app("WindowManager"))
+{
+    create_root_widget();
+    create_frame();
+    create_resolution_list();
+    create_wallpaper_list();
+}
+
+void DisplayPropertiesWidget::create_resolution_list()
+{
+    // TODO: Find a better way to get the default resolution
+    m_resolutions.append({ m_wm_config->read_num_entry("Screen", "Width", 1024), m_wm_config->read_num_entry("Screen", "Height", 768) });
+    m_resolutions.append({ 640, 480 });
+    m_resolutions.append({ 800, 600 });
+    m_resolutions.append({ 1024, 768 });
+    m_resolutions.append({ 1280, 1024 });
+    m_resolutions.append({ 1366, 768 });
+    m_resolutions.append({ 1440, 900 });
+    m_resolutions.append({ 1600, 900 });
+    m_resolutions.append({ 1920, 1080 });
+    m_resolutions.append({ 2560, 1080 });
+
+    m_selected_res = m_resolutions.at(0);
+}
+
+void DisplayPropertiesWidget::create_root_widget()
+{
+    auto* keeper = new GWidget;
```

Comment:
No need for a temporary here, really.
```suggestion
    m_root_widget = new GWidget;
```